docs/fxa-engineering/fxa-dev-process.md

```diff
@@ -147,6 +147,18 @@ Here are some handy questions and things to consider when reviewing code for Fir
       Commits should not be cherry-picked
       between train branches and `master`.
 
+## Browser Support
```

Comment:
👍 to having this somewhere more discoverable. Should we delete the corresponding doc from the fxa repo?